src/Contracts/Http.php

```diff
@@ -46,4 +46,10 @@ public function patch(string $path, $body = null, array $query = []);
      * @throws JsonDecodingException
      */
     public function delete(string $path, array $query = []);
+
+    /**
+     * @throws ApiException
+     * @throws JsonEncodingException
+     */
+    public function postStream(string $path, $body = null, array $query = []): \Psr\Http\Message\StreamInterface;
```

Comment:
This again - misses import statement and using only class name as per coding standards